src/main/java/com/rabbitmq/http/client/Client.java

```diff
@@ -637,12 +637,28 @@ public boolean publish(String vhost, String exchange, String routingKey, Outboun
     return Arrays.asList(this.httpLayer.get(uri, QueueInfo[].class));
   }
 
+  public List<QueueInfo> getQueues(DetailsParameters detailsParameters) {
```

Comment:
Is it worth consolidating the actual implementation to one method, e,.g. `getQueues(String vshot, DetailsParameters detailsParameters)`, and make the others delegate to that one ?